### ISSUE TITLE
Calcolo della funzione di Loss nel punto di minimo

### ISSUE THREAD

**Issue by giomettif** [vision-e/PracticalAI#1]
Ciao Eric,

in relazione all'esercizio 6.

Dopo aver lanciato il modello di regressione logistica, come faccio a calcolare la funzione di Loss (nel punto di minimo) per valutare se sono in overfitting o meno?

Grazie mille

**Comment by vision-e** [comment]
Ciao @giomettif ,

ottima domanda. Ti consiglio di guardare sulla documentazione di scikit-learn qui https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html. 

In generale prova a capire quale ottimizzatore è impostato di default. Cerca di capire se ha senso in questo esempio trovare il valore della funzione di Loss.

Magari poi ne parliamo mercoledì

**Comment by giomettif** [comment]
Ciao Eric,

annoto un'altra serie di punti che mi piacerebbe discutere insieme:
- Nell'esercizio 6 tutte le variabili indipendenti X sono numeriche (float64), cosa succede se alcune sono categoriche e altre numeriche? 
- Significato del parametro C del modello di Logistic Regression ()
- Come associo un certo di Scalar ad un certo tipo di classificatore?
